Apartment.rb

```diff
@@ -0,0 +1,3 @@
+class Apartment < ActiveRecord::Base
+  has_many :tenant
```

Comment:
This should be `has_many :tenants`. A tip is that these methods almost always make sense in terms of english, has_many implies plural, so `:tenants`.